Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -219,17 +243,127 @@ namespace AzFramework
         }
     }
 
+    void XcbNativeWindow::DrawSplash()
+    {
+        auto settingsRegistry = AZ::SettingsRegistry::Get();
+        AZ::SettingsRegistryInterface::FixedValueString splashScreenImagePath;
+        AZ::SettingsRegistryInterface::FixedValueString assetCachePath;
+        static const AZStd::string SplashLogoSetregPath = "/O3DE/xcb/SplashScreenImagePath";
+        static const AZStd::string AssetsSetregPath = "/O3DE/Runtime/FilePaths/CacheProjectRootFolder";
```

Comment:
You cannot have a cache folder predefined in the registry, since it is different with GameLauncher run in install target and YourProject.GameLauncher. Also it will be different for different platforms. I can see you need to get a global path for XCB, but please use a different approach.

Reply:
I currently don't have idea how to do this in clean way so I added this logo binary data using .h header, this still looks ok and header file is only 1.3MB, tho size of the binary data in exe should be much smaller around 300-400KB